google/cloud/security/inventory/inventory_loader.py

```diff
@@ -284,7 +292,12 @@ def main(_):
     configs = FLAGS.FlagValuesDict()
 
     try:
-        pipelines = _build_pipelines(cycle_timestamp, configs, dao)
+        pipelines = _build_pipelines(
+            cycle_timestamp,
+            configs,
+            dao=dao,
+            project_dao=project_dao,
+            organization_dao=organization_dao)
```

Comment:
I don't mind :) But am wondering if you've already done the work? I don't want to step on your toes there.